django_orm_views/not_managed_model.py

```diff
@@ -0,0 +1,46 @@
+import copy
+
+from django.db.models import Model
+
+from .constants import SUB_SCHEMA_NAME
+
+
+class NotManagedModel(Model):
+    """
+    Abstract model used to define not managed django models that use
+    database views as a data source.
+
+    Warning: adding custom Meta class to the child model leads to an error
+    """
+    name = None
+
+    class Meta:
+        abstract = True
```

Comment:
Do we even need this anymore?

Reply:
We don't need the name, but the Meta is needed because it should be an abstract Model, the children inheriting from it will not be abstract though

Reply:
Done